tests/test_scene_rendering/infallible_scenes.py

```diff
@@ -1,13 +1,18 @@
 from __future__ import annotations
 
-from manim import *
+from manim import Scene, Square
 
 
 class Wait1(Scene):
     def construct(self):
         self.wait()
 
 
+class Wait3(Scene):
```

Comment:
this seems unrelated?

Reply:
Have you taken a look at the issues?
The thing was that if such a thing ran on main now then the output you get is not expected, `Wait1` will be a video but `Wait3` and `Wait2` will be images, whereas `Wait2` should actually be a video.

So, this is kind of a test to check that 😓

Reply:
Now I actually understand -- also I appreciate that you changed the order of the test scenes. :-)